api/src/main/java/org/openmrs/module/rwandaemr/integration/CitizenProvider.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.rwandaemr.integration;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.http.HttpEntity;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.util.EntityUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Supports connections to and operations with the citizen endpoints in the HIE
+ */
+@Component("citizenProvider")
+public class CitizenProvider {
+
+	protected Log log = LogFactory.getLog(getClass());
+
+	private final IntegrationConfig integrationConfig;
+
+	public CitizenProvider(
+			@Autowired IntegrationConfig integrationConfig
+	) {
+		this.integrationConfig = integrationConfig;
+	}
+
+	/**
+	 * This will attempt to retrieve the Citizen from the Citizen from the HIE population registry
```

Comment:
typo? or is it really the Citizen from the Citizen>

Reply:
Fixed, thanks